src/digilib-iiif-migrate.js

```diff
@@ -10,6 +10,33 @@ function getIiifScaler(digilibUrl) {
 }
 
 /**
+ * Check whether the provided url is a digilib scaler URL
+ * @param {string} digilibUrl - digilib URL that should be checked
+ * @returns {boolean}
+ */
+function isDigilibScaler(digilibUrl) {
+  return digilibUrl.includes('Scaler');
```

Comment:
i'd say this should be more strict, not necessarily by using a regex, but the enclosing slashes would decrease the uncertainty to hit a false positive sequence sigificantly with minor effort.